_pages/rfc-pairing.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pageminimal
+title: "RFC [r] - WeAllJS #pairing"
+permalink: /rfc-pairing
+---
+
+# RFC [r] - WeAllJS #pairing
+
+[ratified 2016-09-17]
+{: text-italic}
+
+Learning is cool! Programming is hard! Learning with someone else is often way more productive than trying to learn alone.
+We All JS  strives to be a community where people learning something new, no matter their level of skill can find support.
+Here at We All JS, we recommend two different forms of cooperative programming.
+
+## Code Review
+Code review is an extremely useful medium for cooperative learning. A more experienced person code reviewing code from a
```

Comment:
How about just "A more experienced person reviewing code from a newcomer..." The duplicate 'code' makes this sentence clunky.